.github/workflows/johnnytest4.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+output_file="CMake_Error_Log.txt"
+
+# Clean the output file first
+echo "" > $output_file
```

Comment:
That seems ok, but if you rerun the CMake, next time this overwrite the previous log.
You may considering some things like:
- do we need the previous content of CMake_Erro_Log.txt is we re-run CMake?
- if yes then how to keep the previous content? (maybe adding a timestamp to the file name using date command?)
- if it is a log file, we usually using then '.log' as the extension.